lib/membrane_live_web/channels/event_channel.ex

```diff
@@ -163,17 +165,17 @@ defmodule MembraneLiveWeb.EventChannel do
   # and sends message back) -> server (updates presenter in Presence)
   # such design is caused by user Presence that can be updated only with its socket-channel combination
   # (socket parameter in function below)
-  def handle_in("presenter_remove", %{"presenter" => presenter}, socket) do
-    {:ok, _ref} =
-      Presence.update(socket, presenter, fn map -> Map.put(map, :is_presenter, false) end)
+  def handle_in("presenter_remove", %{"email" => email}, socket) do
+    {:ok, _ref} = Presence.update(socket, email, fn map -> Map.put(map, :is_presenter, false) end)
 
     "event:" <> id = socket.topic
-    remove_from_presenters(presenter, id)
+    remove_from_presenters(email, id)
     {:noreply, socket}
   end
 
-  def handle_in("presenter_remove", %{"presenter_topic" => presenter_topic}, socket) do
-    props = Presence.get_by_key(socket, List.last(String.split(presenter_topic, ":")))
+  def handle_in("presenter_remove", %{"presenterTopic" => presenter_topic}, socket) do
+    presenter_email = List.last(String.split(presenter_topic, ":"))
```

Comment:
According to the style guide, which we follow in Membrane you should use pipe in situations like this.
https://github.com/christopheradams/elixir_style_guide#pipe-operator